src/controllers/dynakube/deploymentmetadata/config.go

```diff
@@ -0,0 +1,28 @@
+package deploymentmetadata
+
+import (
+	"github.com/Dynatrace/dynatrace-operator/src/logger"
+)
+
+const (
+	orchestrationTech = "Operator"
+
+	keyOperatorScriptVersion = "script_version"
+	keyOrchestratorID        = "orchestrator_id"
+	keyOrchestrationTech     = "orchestration_tech"
+
+	DeploymentTypeApplicationMonitoring = "application_monitoring"
+	DeploymentTypeFullStack             = "classic_fullstack"
+	DeploymentTypeCloudNative           = "cloud_native_fullstack"
+	DeploymentTypeHostMonitoring        = "host_monitoring"
+	DeploymentTypeActiveGate            = "active_gate"
+
+	EnvDtDeploymentMetadata = "DT_DEPLOYMENT_METADATA"
+
+	OneAgentMetadataKey   = "oneagent"
```

Comment:
`key...` (with `keyOrchestratorID`) as prefix or `...Key` (with `OneAgentMetadataKey`) as suffix should be consistent, both versions are used here